docs/src/pages/index.tsx

```diff
@@ -183,44 +264,120 @@ function Features() {
+      />
+
+      <Feature
+        text="Audio support"
+        secondaryText="Mix audio directly in LiveCompositor."
+        image={<MdAudiotrack className={styles.featureIcon} style={{ fontSize: 80 }} />}
+      />
+
+      <Feature
+        text="WebGPU and Rust"
+        secondaryText="Leverage Rust and WebGPU rendering for great performance."
         image={wgpuLogo}
       />
+
       <Feature
-        text="Real-time processing"
-        secondaryText="Process live video streams in real-time with low latency. Blazingly fast processing makes video conferencing, live-streaming, and broadcasting an everyday bread and butter for Live Compositor."
-        image={<MdLiveTv className={styles.featureIcon} style={{ fontSize: 80 }} />}
-        inverted
+        text="Animated transitions"
+        secondaryText="Change layouts on the fly with animated transitions."
+        image={<GiFeather className={styles.featureIcon} style={{ fontSize: 80 }} />}
       />
+
       <Feature
-        text="Static content"
-        secondaryText="Render text and static images onto your output streams or pass them to other components for further processing."
-        image={<FaImage className={styles.featureIcon} style={{ fontSize: 80 }} />}
+        text="Real-time processing"
+        secondaryText="Process live streams in real-time with low latency."
+        image={<MdLiveTv className={styles.featureIcon} style={{ fontSize: 80 }} />}
       />
+
       <Feature
-        text="Web rendering"
-        inverted
-        secondaryText="You can render any website and combine it with video streams or other elements using the Chromium browser embedded inside the compositor."
-        image={<FaChrome className={styles.featureIcon} style={{ fontSize: 80 }} />}
+        text="Offline processing"
+        secondaryText="Use LiveCompositor for non-real-time use cases with offline processing mode."
+        image={<IoCloudOffline className={styles.featureIcon} style={{ fontSize: 80 }} />}
       />
     </div>
   );
 }
 
-function IntegrationMembranePlugin() {
+function StandaloneSever() {
+  return (
+    <div className="row" style={{ justifyContent: 'center' }}>
+      <FaServer className="margin--lg" style={{ fontSize: 150, alignSelf: 'center' }} />
+      <div className="col">
+        <div className="card">
+          <div className="card__header">
+            <Heading as="h3" className="margin-top--sm">
+              Standalone server
+            </Heading>
+          </div>
+          <div className="card__body container">
+            <p>
+              LiveCompositor can be deployed as a standalone server. Language-agnostic API allows
+              you to seamlessly integrate it into your existing solutions. Docker images and
+              deployment section in docs are meant to make the deployment process as easy as
+              possible.
+            </p>
+            <div className="row margin--sm" style={{ justifyContent: 'center' }}>
```

Comment:
- I don't think this looks good
  - centering those buttons on large screens looks weird (it looks fine on small screens)
  - I'm not convinced that you want to style the differently (but I also don't object if you want)
  - They fill too large, too much out of place compared to other elements next to it.
  
It's hard to say what to do without seeing how it will look, but I would suggest changing alignment to left or right, and keeping minimal width based on content instead of stretching it static width. The same as about us card